holoviews/operation/datashader.py

```diff
@@ -176,6 +194,15 @@ def _process(self, element, key=None):
         category = agg_fn.column if isinstance(agg_fn, ds.count_cat) else None
         x, y, data, glyph = self.get_agg_data(element, category)
 
+        if x is None or y is None:
+            x0, x1 = self.p.x_range or (-0.5, 0.5)
+            y0, y1 = self.p.y_range or (-0.5, 0.5)
```

Comment:
Seems like unit range is the default. Guess that is fine as long as this is sensible default behavior when the data is missing.